pkg/build/buildutil/util_test.go

```diff
@@ -8,6 +8,32 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+func TestLabelValue(t *testing.T) {
+	testCases := []struct {
+		name           string
+		input          string
+		expectedOutput string
+	}{
+		{
+			name:           "do-not-end-with-hyphen",
+			input:          "cluster-kube-controller-manager-operator-4.3.0.ipv6-2019-11-27-0001-build",
+			expectedOutput: "cluster-kube-controller-manager-operator-4",
```

Comment:
I don't see this test case ending with a hyphen?

Reply:
Yeah this is the value from the bug ... the 63rd character is a hyphen